packages/react-app/src/styles/edit.scss

```diff
@@ -65,6 +65,21 @@
   color: #cc6161;
 }
 
+.image-input-filled {
+  width: 100%;
+  max-height: 250px;
+  cursor: pointer;
```

Comment:
Maybe I am missing something, but is the image clickable?

Reply:
Yes, it renders 2 different components based on if theres as image uploaded or not, but it is always clickable 👍